src/lib/components/workflow/filter-search/datetime-filter.svelte

```diff
@@ -141,17 +137,13 @@
   />
   <MenuContainer>
     <MenuButton
+      unroundLeft
       id="time-range-filter"
       controls="time-range-filter-menu"
-      class="flex flex-row items-center p-2 bg-white text-gray-800 hover:border-primary hover:bg-primary hover:text-white border border-gray-800 rounded-r h-10"
     >
       {translate('workflows', 'select-time')}
     </MenuButton>
-    <Menu
-      keepOpen
-      id="time-range-filter-menu"
-      class="flex rounded h-auto w-[400px] flex-col gap-8 bg-white p-2"
-    >
+    <Menu keepOpen id="time-range-filter-menu" class="w-[400px] p-2">
```

Comment:
Not sure how we want to handle dropdowns within dropdowns, but not being able to see the content unless you scroll definitely ins't ideal.

| Before | After |
|--|--|
|![Screenshot 2023-08-18 at 11 25 27 AM](https://github.com/temporalio/ui/assets/15069288/5a15a384-7ac6-40fa-9105-31dd695d1c2f)|![Screenshot 2023-08-18 at 11 26 09 AM](https://github.com/temporalio/ui/assets/15069288/7d41ff45-ceb8-4f72-8c07-ab9b81dd56d6)|